Pod/Classes/Optional Data Sources/VOKDefaultPagingAccessory.m

```diff
@@ -22,13 +22,11 @@ - (void)didMoveToSuperview
         self.clipsToBounds = YES;
         [self setBackgroundColor:[UIColor clearColor]];
         
-        UILabel *label = [[UILabel alloc] initWithFrame:(CGRect){CGPointZero, self.frame.size}];
+        UILabel *label = [[UILabel alloc] initWithFrame:CGRectMake(0, 0, self.frame.size.width, self.frame.size.height)];
```

Comment:
`(CGRect){ .origin = CGPointZero, .size = self.frame.size }` would be fine, too.  🍅 :tomayto::tomahto: